src/Orc.Controls/Orc.Controls.Shared/Controls/LogViewer/Views/LogViewerControl.xaml.cs

```diff
@@ -216,6 +216,17 @@ public bool AutoScroll
             typeof(LogViewerControl), new FrameworkPropertyMetadata(true, FrameworkPropertyMetadataOptions.BindsTwoWayByDefault));
 
 
+        [ViewToViewModel(MappingType = ViewToViewModelMappingType.TwoWayViewWins)]
+        public bool ShowMultilineMessageExpanded
```

Comment:
I think we need to rename this to "ShowMultilineMessagesExpanded" (since this applies to all messages)